_ping.php

```diff
@@ -1193,6 +1193,21 @@ protected function check2(): void {
       if (filesize($file) !== 0) {
         continue;
       }
+
+      $mtime = filemtime($file);
+      if (!$mtime) {
+        $this->setStatus('error', 'Could not get mtime of the file in the public files directory.', [
+          'file' => $file,
+        ]);
+        return;
+      }
+
+      // Do not clean up fresh files.
+      // In the multicontainer environment parallel pings would kill each other.
+      if ($mtime > time() - 3600) {
```

Comment:
Is 1hour too much time? Maybe 5mins would be better? If every call creates a test file, and there are multiple calls every second, there's going to be hell of a lot of test files laying around 😂

Reply:
We have the "FsSchemeDeleteChecker" that should delete the files 
and the cleanup only handles files that are "left behind"
in most  cases this should not happen 

but when ping is used "too frequently" with multi pod drupal the "cleanup" will detect files that are not yet deleted by the "deleteChecker" from other ping requests from other pods

so there should not be a huge amount of undeleted files this way